WhatsNewInCTSM5.3.md

```diff
@@ -1,53 +1,48 @@
-in: CN Matrix to speed up spinup for the BGC model, updated surface datasets, updated Leung 2023 dust emissions,
-explicit Air Conditioning for the Urban model, updates to crop calendars. For clm6_0 physics these options are now
-default turned on in addition to Sturm snow, and excess ice.
+## Purpose and description of changes since CTSM 5.2 (tag `ctsm5.2.005`)
+- Adds CN Matrix method to speed up spinup for the BGC model.
+- Updates surface datasets.
+- Brings in new Leung 2023 dust emissions.
+- Adds explicit air conditioning for the urban model.
+- Updates crop calendars.
+- Updates fire model with various improvements, including default parameterization against climate reanalysis from CRU-JRA instead of GSWP.
+- FATES compsets can now be run with transient land use.
 
-Changes to CTSM Infrastructure:
-===============================
+These changes were needed ahead of the CESM3 capability/functionality "chill". For `clm6_0` physics these options are now turned on by default, in addition to Sturm snow and excess ice.
 
- - manage_externals removed and replaced by git-fleximod
- - Ability to handle CAM7 in LND_TUNING_MODE
+## Changes to CTSM infrastructure
+ - `manage_externals` removed and replaced by `git-fleximod`
+ - Ability to handle CAM7 in `LND_TUNING_MODE`
 
-Changes to CTSM Answers:
-========================
+## Changes to CTSM answers
 
- Changes to defaults for clm6_0 physics:
+ Changes to defaults for `clm6_0` physics:
   - Urban explicit A/C turned on
-  - Snow thermal conductivity is now Sturm_1997
-  - New IC file for f09 1850
+  - Snow thermal conductivity method is now `Sturm_1997`
+  - New initial conditions file for f09 ("1-degree") 1850, with more in progress
   - New crop calendars
-  - Dust emissions is now Leung_2023
+  - Dust emissions method is now `Leung_2023`
   - Excess ice is turned on
-  - Updates to MEGAN for BVOC's
+  - Updates to MEGAN for BVOCs
   - Updates to BGC fire method
 
  Changes for all physics versions:
-
   - Parameter files updated
   - FATES parameter file updated
   - Glacier region 1 is now undefined
-  - Update in FATES transient Land use
+  - Update in FATES transient land use
   - Pass active glacier (CISM) runoff directly to river model (MOSART)
-  - Add the option for using matrix for Carbon/Nitrogen BGC spinup
+  - Add the option for using Matrix CN method for Carbon/Nitrogen BGC spinup
 
-New surface datasets:
-=====================
+## New surface datasets
 
-- With new surface datasets the following GLC fields have region "1" set to UNSET:
-     glacier_region_behavior, glacier_region_melt_behavior, glacier_region_ice_runoff_behavior
+- With new surface datasets the following GLC fields have region "1" set to UNSET: glacier_region_behavior, glacier_region_melt_behavior, glacier_region_ice_runoff_behavior
 - Updates to allow creating transient landuse timeseries files going back to 1700.
```

Comment:
So `mksurfdata_esmf` can now make land-use timeseries files back to 1700. Have any of those actually been made?